integration/release/release.go

```diff
@@ -0,0 +1,61 @@
+// +build k8srequired
+
+package release
+
+import (
+	"log"
+	"time"
+
+	"github.com/cenkalti/backoff"
+	"github.com/giantswarm/e2e-harness/pkg/framework"
+	"github.com/giantswarm/helmclient"
+	"github.com/giantswarm/microerror"
+)
+
+func WaitForReleaseStatus(gsHelmClient *helmclient.Client, release string, status string) error {
+	operation := func() error {
+		rc, err := gsHelmClient.GetReleaseContent(release)
```

Comment:
Is the helm client not retrying? If not, we should make a backoff configurable there and use it that way. WDYT?

Reply:
The helm client does indeed retry, I will rework this in the current test. Nice catch.

https://github.com/giantswarm/helmclient/blob/b48ae48e60b4891baf6f3ab23eff9ed469532ad0/helmclient.go#L270

Reply:
Nevermind, we retry in the helm client and get a response from there. In this function we are waiting for a _specific_ response and then retry here again with a timeout until we get the specific response. 
So this is fine imo